tests/component/_task_modules/channels/test_ai_channels.py

```diff
@@ -0,0 +1,722 @@
+import pytest
+
+from nidaqmx import Task
+from nidaqmx._task_modules.channels.ai_channel import AIChannel
+from nidaqmx.constants import (
+    AccelChargeSensitivityUnits,
+    AccelSensitivityUnits,
+    AccelUnits,
+    ACExcitWireMode,
+    BridgeConfiguration,
+    BridgeUnits,
+    ChargeUnits,
+    CJCSource,
+    CurrentShuntResistorLocation,
+    EddyCurrentProxProbeSensitivityUnits,
+    ExcitationSource,
+    ForceIEPESensorSensitivityUnits,
+    ForceUnits,
+    LVDTSensitivityUnits,
+    ResistanceConfiguration,
+    RTDType,
+    StrainGageBridgeType,
+    StrainGageRosetteMeasurementType,
+    StrainGageRosetteType,
+    TemperatureUnits,
+    TerminalConfiguration,
+    ThermocoupleType,
+    UsageTypeAI,
+    VoltageUnits,
+)
+
+
+# Note: Tests for other channel types will be less complete given that the underlying Python
+# implementation is code-generated and the underlying C API implementation is well unit-tested.
+@pytest.mark.parametrize(
+    "desired_term_config, expected_term_cfg",
+    [
+        (TerminalConfiguration.DEFAULT, TerminalConfiguration.DIFF),
+        (TerminalConfiguration.DIFF, TerminalConfiguration.DIFF),
+        (TerminalConfiguration.RSE, TerminalConfiguration.RSE),
+    ],
+)
+@pytest.mark.parametrize("min_val, max_val", [(-10, 10), (-5, 5)])
+@pytest.mark.parametrize(
+    "units, custom_scale_name",
+    [(VoltageUnits.VOLTS, ""), (VoltageUnits.FROM_CUSTOM_SCALE, "no_scaling_scale")],
+)
+def test___ai_channel_collection___add_ai_voltage_chan___sets_channel_attributes(
```

Comment:
You don't necessarily need to put the SUT name in the given clause, since that is covered by the test module name. It's ok to use "test___valid_task___" or "test___task___" since that the real prerequisite is that you created a task. I guess the device is also a prereq but that is true for 99% of tests so it's not interesting unless you call out what kind of device it is.